jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/BaseImageParameters.java

```diff
@@ -34,34 +36,31 @@ public BaseImageParameters(ObjectFactory objectFactory, String imageDescriptor)
     auth = objectFactory.newInstance(AuthParameters.class, imageDescriptor + ".auth");
   }
 
+  @Input
   @Nullable
-  @Override
   public String getImage() {
```

Comment:
I think we still need the `@Optional` on each of these methods, right?

Reply:
Good catch. Seems like `getCredHelper` and `auth` should be optional too?

Reply:
Yeah, in both classes; didn't want to spam the same comment on each method haha